Week3/homework/step2-2.js

```diff
@@ -2,19 +2,27 @@
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
-
-  // Replace this comment and the next line with your code
-  console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  for (let i = startIndex; i <= stopIndex; i++) {
+    numbers.push(i);
+  }
+  for (let i = 0; i < numbers.length; i++) {
+    if (numbers[i] % 3 === 0 && numbers[i] % 5 === 0) {
```

Comment:
This is a correct solution, I believe it's as a 95% to 97% correct, looking back to the question it said:
> Both functions should be called if the array value is divisible by both 3 and 5.

and that's why it's correct.
the `if` block is protecting any code to be executed by the next `else if` statements,
that means we have
 - `if` it's divisible by 5 and 3 do this
 - `else if` it's divisible by 3 do that
 - `else if` it's divisible by 5 do the other

if we removed the else statements there is nothing preventing the function from executing the rest, what if we could focus on the result of executing both the functions on 3 and 5, so perhaps we only need two `if` statements in this case? 🙂